gps_rviz/src/gps_rviz.cpp

```diff
@@ -1,152 +1,63 @@
 #include "ros/ros.h"
 #include "sensor_msgs/NavSatFix.h"
 #include "geometry_msgs/Vector3.h"
-#include "visualization_msgs/Marker.h"
-#include "math.h"
-
+#include "gps_handler.cpp"
+#include "gps_rviz.h"
```

Comment:
We use this include style: https://github.com/ethz-asl/multiagent_mapping/wiki/Header-include-guide
Alphabetize includes.